### DISCUSSION TITLE
Livewire Best Practices Update

### DISCUSSION THREAD

**Discussion by michael-rubel** [livewire/livewire#6793] [2. Show and tell]
Hello everyone! 👋

I'm working on the best practices update to meet the new reality of Livewire v3:
https://github.com/michael-rubel/livewire-best-practices/pull/9

At the moment I haven't tried all the new features. Thus, there might be some mistakes.

I'm asking for a review of the PR, so we can fix issues (if we have any) or improve things before merging it to `main` and fully finalizing the shift to v3.

**Comment by anhofmann** [comment]
Great writeup, thank you for this!

The "Keep track of a DOM elements" part isn't right, I think.
Correct would be something like "you do not need `wire:key` anymore, except when loading livewire components in a loop, then `key` is mandantory." 
https://livewire.laravel.com/docs/nesting#rendering-children-in-a-loop

**Reply by michael-rubel** [reply]
@anhofmann Yeah, removed this one completely.

**Comment by michael-rubel** [comment]
Merged the PR. If you see any issues, PRs are welcome!
